prep-css-box-model/styles.css

```diff
@@ -0,0 +1,17 @@
+body {
+  text-align: center;
+}
+
+button {
+  background-color: red;
+  color: white;
+  margin: 20px;
+  padding: 10px;
+}
+
+/* 👇 YOUR CODE HERE 👇 */
+p.double {
+  border-style: double;
+  color: darkgray;
+  width: "3x";
```

Comment:
Try moving these properties into your rule set for `button` above. Your buttons should look like this with a thin double border.

<img width="767" alt="image" src="https://user-images.githubusercontent.com/7432943/172965426-ccab4256-d826-4afa-a6fe-dab160bfd29c.png">